etna/transforms/feature_selection/gale_shapley.py

```diff
@@ -251,8 +252,10 @@ class to build relevance table
             if "all" value is given, all columns are used
         use_rank:
             if True, use rank in relevance table computation
+        return_features:
+            bool, indicates whether to return features or not.
```

Comment:
Make the description here consistent with the descriptions in the other files(remove bool)